### ISSUE TITLE
vue event with @cell-contextmenu is stopPropagation in cell

### ISSUE THREAD

**Issue by ginkosen** [antoniandre/vue-cal-v4#343]
**outside the vue-cal we need contextmenu event Propagation**

```js
onCellContextMenu (DOMEvent) {
      DOMEvent.stopPropagation()
      DOMEvent.preventDefault()
      const date = new Date(this.data.startDate)
      const { cursorCoords, minutes } = this.vuecal.minutesAtCursor(DOMEvent)
      date.setMinutes(minutes)
      // If splitting days, also return the clicked split on cell contextmenu when emitting event.
      const split = this.splitsCount ? this.getSplitAtCursor(DOMEvent) : null
      this.vuecal.$emit('cell-contextmenu', { date, ...cursorCoords, ...(split || {}) })
    }
```

### expect 

**don't stopPropagation** 

### solution

add cell-contextmenu-propagation prop or ...

**Comment by antoniandre** [comment]
Hi @ginkosen,

Sorry for the delay. I have released version 3.8.3 where the `cell-contextmenu` also returns the original DOM event.

The `DOMEvent.preventDefault()` that you pointed is intended to prevent the browser context menu. But it doesn't stop you from using the context menu and defining a handler yourself via `@cell-contextmenu`.

Note that this event is only triggered when the `cell-contextmenu` prop is set to true (false by default). I realized it was missing in the documentation's API section so I updated it https://antoniandre.github.io/vue-cal/#api.

Hope it helps.

If that's not enough for your use case, please explain what you want to do and provide some code or ideally a reproduction link.

**Comment by ginkosen** [comment]
thanks for your smart replay and idea !!!
